src/Tasks.php

```diff
@@ -923,6 +926,7 @@ private function getDatabaseOfTruth() {
     }
 
     if ($getDB) {
+      $this->taskExec('drush sql:create -y')->dir($project_properties['web_root'])->run();
```

Comment:
@rjacobsen0 this command is perfect! But it does look like it also run the sql:drop below, so that command is now redundant. Let's just replace 

```
$this->taskExec('drush sql:drop -y')->dir($project_properties['web_root'])->run();
```
with 
```
$this->taskExec('drush sql:create -y')->dir($project_properties['web_root'])->run();
```
instead.